### PR TITLE
Update en-US, fi and ru locales. JB#56342 OMP#JOLLA-540

### PR DESCRIPTION
**Pull request by llewelld**
Updated using the `mine-localisation-files.py` tool.

### CODE REVIEW

**Comment by llewelld** [comment]
The en-US files are taken directly from the ESR78 repository (the mobile/Android versions of the files).

**Comment by llewelld** [comment]
Switching to non-draft. There may still be some outstanding issues related to the certificate ids, but I'll create a separate task for that.

**Comment by llewelld** [comment]
Rebased.

**Comment by llewelld** [comment]
Example translations with the changes applied.

translations

File: overrides/en-US/netError.dtd [hunk 1]
```diff
@@ -77,15 +77,6 @@
 <!ENTITY notCached.longDesc "<p>The requested document is not available in &brandShortName;’s cache.</p><ul><li>As a security precaution, &brandShortName; does not automatically re-request sensitive documents.</li><li>Click Try Again to re-request the document from the website.</li></ul>">
 
 <!ENTITY netOffline.title "Offline mode">
-<!-- LOCALIZATION NOTE (netOffline.longDesc3) This string contains markup including widgets enabling wifi connections.
-     The text inside the tags should be localized. Do not touch the ids. -->
-<!ENTITY netOffline.longDesc3 "
-<ul>
-  <li>Try again. &brandShortName; will attempt to open a connection and reload the page.
-    <button id='wifi'>Enable Wi-Fi</button>
```
[the remark this answers is not in the document]
Comment (llewelld) [reply overrides/en-US/netError.dtd:85]: Ah, nicely spotted. It turns out we do still have this button, but the string is provided by [`connectionFailure.longDesc2`](https://github.com/sailfishos/embedlite-components/blob/38554f11da5d1ff33fabc459b696085f3bf94021/overrides/en-US/netError.dtd#L14) which aliases [`sharedLongDesc3`](https://github.com/sailfishos/embedlite-components/blob/38554f11da5d1ff33fabc459b696085f3bf94021/overrides/en-US/netError.dtd#L166) instead.

The [associated mozilla bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1581204) about removing this sting isn't especially helpful for finding out why this change was made I'm afraid.

Screenshots:


unable-to-connect

**Review by rainemak — APPROVED** [review]
LGTM, any thoughts @okodron ?

**Review by okodron — APPROVED** [review]

**Comment by okodron** [comment]
> LGTM, any thoughts @okodron ?

LGTM as well.

**Pull request merged**